apps/condo/domains/common/components/ChangeHistory/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { green } from '@ant-design/colors'
+import { Col, Row, Skeleton, Button } from 'antd'
+import React, { ComponentProps, ReactElement, useCallback, useMemo, useState } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Typography } from '@open-condo/ui'
+
+import { fontSizes } from '@condo/domains/common/constants/style'
+
+import { BaseChangesType, HistoricalChange } from './HistoricalChange'
+
+
+type ChangeHistoryInputType<ChangesType> = {
+    items: ChangesType[]
+    total: number
+    loading: boolean
+    title: string
+    useChangedFieldMessagesOf: ComponentProps<typeof HistoricalChange>['useChangedFieldMessagesOf']
+    Diff: ComponentProps<typeof HistoricalChange>['Diff']
+}
+
+type ChangeHistoryReturnType = ReactElement | null
+
+const CHANGES_PER_CHUNK = 5
+const TEXT_BUTTON_STYLE: React.CSSProperties = {
```

Comment:
This stuff can be done with ui-kit button and Typography

Reply:
This is an old component. Didn't find a way to do it with new ui kit so left the existing solution